File: smp2017-android-muskan/cs50_psets/pset5/pset6/Muskan2108-cs50-2017-x-pset6/greedy.py
import cs50
c=0
print("O hai! How much change is owed?")
f=cs50.get_float()
while True:
    if f<0:
        print("How much change is owed?")
        f=cs50.get_float()
    else:
        break
t=int(round(f*100))
s=t-(t%25)
c+=s/25
t=t-s
s=t-(t%10)
c+=s/10
t=t-s
s=t-(t%5)
c+=s/5
t=t-s
c+=t
print("{}".format(c))

File: smp2017-android-aanchal/cs50  pset/pset1/AanchalHanda-cs50-2017-x-greedy/greedy.c
#include<stdio.h>
#include<cs50.h>
#include<math.h>
int main(void)
{  printf("O hai! How much change is owed?\n");
   float ch=get_float();
   while(ch<0)
   {printf("O hai! How much change is owed?\n");
    ch=get_float();}
  
   int change=round(ch*100);
   int c=0;
   
   c=change/25;
        change=change%25;
        
         c+=change/10;
        change=change%10;
       
         c+=change/5;
        change=change%5;
      
        c+=change;
       printf("%i\n",c);
    }